docs/self_hosting/configuration/scale.mdx

```diff
@@ -1,41 +1,212 @@
 
-The table below contains high level guidance for LangSmith configurations, wheres cale is measured in terms of traces per second (TPS). The configuration recommendations also assume that you are using a relatively new version of the SDK and LangSmith (at least `0.10.69`). It is strongly recommended to run high load against a Kubernetes deployment of LangSmith.
+For example configurations, check out our [scale examples](#example-langsmith-configurations-for-scale)
 
-| Desired traces per second (TPS)         | Queue pod replicas | Frontend replicas | Platform backend replicas | ClickHouse config               | Redis cache size |
-| --------------------------------------- | ------------------ | ----------------- | ------------------------- | ------------------------------- | ---------------- |
-| [**10 TPS**](#10-traces-per-second-tps) | 3 (default)        | 1 (default)       | 3 (default)               | 4 vCPUs, 16 GB Memory (default) | 2 GB (default)   |
-| [**100 TPS**](#100-tps)                 | **10**             | **2**             | 3 (default)               | 4 vCPUs, 16 GB Memory (default) | **13+ GB**       |
-| [**1000 TPS**](#1000-tps)               | **170**            | **4**             | **20**                    | **10 vCPUs, 48 GB Memory**      | **200+ GB**      |
+## Trace ingestion (write path)
 
-:::note
-Your exact usage pattern of the product may require more tuning of resources. If that is the case, please reach out to the LangSmith team for any questions.
+Services that play a large role in trace ingestion:
+
+- Platform backend service: Receives initial request to ingest traces and places traces on a Redis queue
+- Redis cache: Used to queue traces that need to be persisted
+- Queue service: Persists traces for querying
+- ClickHouse: Persistent storage used for traces
+
+Common usage that put load on the write path:
+
+- Ingesting traces via the Python or JavaScript LangSmith SDK
+- Ingesting traces via the `@traceable` wrapper
+- Submitting traces via the `/runs/multipart` endpoint
+
+When scaling up the write path (trace ingestion), it is helpful to monitor the four services/resources listed above. Here are some typical changes that can help increase performance of trace ingestion:
+
+- Give ClickHouse more resources (CPU and memory) if it is approaching resource limits.
+- Increase the number of platform-backend pods if ingest requests are taking long to respond.
+- Increase queue service pod replicas if traces are not being processed from Redis fast enough.
+- Use a larger Redis cache if you notice that the current Redis instance is reaching resource limits. This could also be a reason why ingest requests take a long time.
+
+## Trace querying (read path)
```

Comment:
If you go for the different structure pattern for that write path section, I would replicate that here!